src/lib/filters/scraps/scrapPing.go

```diff
@@ -26,12 +26,25 @@ func CLIPingScrap(s *string) (a *TypePingScrap) {
 		if words[0] == "rtt" {
 			temp := strings.Split(words[len(words)-2], "/")
 			a = &TypePingScrap{
-				Min: strToFloat64(temp[0]),
-				Avg: strToFloat64(temp[1]),
-				Max: strToFloat64(temp[2]),
+				Min:  strToFloat64(temp[0]),
+				Avg:  strToFloat64(temp[1]),
+				Max:  strToFloat64(temp[2]),
 				Mdev: strToFloat64(temp[3]),
 			}
 		}
 	}
 	return
-}
\ No newline at end of file
+}
+
+// CLIFLoodPingScrap returns packet loss
+func CLIFLoodPingScrap(s *string) (a uint64) {
```

Comment:
`a` should be of type `float32` since the packet loss us always of a decimal value. Please change it accordingly in all respective places